src/test/java/duke/task/DeadlineTest.java

```diff
@@ -0,0 +1,15 @@
+package duke.task;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class DeadlineTest {
+    @Test
+    public void event_undoneToString_correctOutput() {
```

Comment:
is the method name suppose to be "**deadline**_undoneToString_correctOutput()"?

It seems to be having the same name as the test method in EventTest.